src/renderer/views/app/components/Tabbar/style.ts

```diff
@@ -16,6 +17,9 @@ export const StyledTabbar = styled.div`
   margin-left: 4px;
   margin-right: 32px;
   display: flex;
+  ${({ isFullscreen }: { isFullscreen: boolean }) => css`

```

Comment:
The `isFullscreen` prop should be passed there, like that:
https://github.com/wexond/desktop/blob/25ac0ae8eb0c7ef02a9381a9e0f7cd079726527a/src/renderer/views/app/components/NavigationButtons/index.tsx#L40

Reply:
Just pushed commit 3c93bc5808e7fb04521a41a1485c73d88927469e which resolves this issue